docs-only: large fixes and template troubleshooting section (#519)
nautobot/nautobot-plugin-golden-config: docs/user/troubleshooting/troubleshoot_dispatchers.md

# Troubleshooting Dispatchers

> At a high-level the default dispatchers that Golden Config uses are actually housed in another open source repository. [nornir-nautobot](https://docs.nautobot.com/projects/nornir-nautobot/en/latest/) holds the Nornir tasks that define the methods that Golden Config utilizes.

Reviewer: ```suggestion
At a high-level the default dispatchers that Golden Config uses are actually sourced from another open source library. [nornir-nautobot](https://docs.nautobot.com/projects/nornir-nautobot/en/latest/) contains the Nornir tasks that define the methods that Golden Config utilizes.
```
